docs/QuickstartDeployPortal.md

```diff
@@ -28,20 +28,25 @@ Below is a table that describes all of the parameters in the custom deployment t
 |Enable Aad Smart on Fhir Proxy|**Yes**|Set to true to enable the [AAD Smart on FHIR Proxy](https://docs.microsoft.com/en-us/azure/healthcare-apis/use-smart-on-fhir-proxy)|
 |Msdeploy Package Url|No|Webdeploy package to use as depoyment code. If blank, the latest code package will be deployed.|
 |Deploy Application Insights|**Yes**|If this is set to true, it enables logging|
 |Application Insights Location|**Yes**|Location for Application Insights|
 |Additional Fhir Server Config Properties|No|You can specify changes to the App Settings json file. Note that you can also make these changes after deployment|
-|Solution Type|**Yes**|Currently this is set to deploy SQL and cannot be changed. If you would like to deploy CosmosDB instead, you can do that [here](https://github.com/microsoft/fhir-server/blob/master/samples/templates/default-azuredeploy.json)|
-|Sql Admin Password|**Yes**|This is the admin password for your SQL Server. Note that there are some password requirements that you can read about [here](https://docs.microsoft.com/en-us/sql/relational-databases/security/password-policy?view=sql-server-ver15#password-complexity)|
-|Sql Location|**Yes**|This is set by default to deploy to the same region as the web application. If you set this to a different application, you will incure charges since your web application will be in one region and your server will be in another|
-|Fhir Version|**Yes**|Select the FHIR version (STU3, R4, or R5) that you want your FHIR Server to be on.|
-|Enable Export|No|Determines whether export will be enabled for this fhir instance. If true, a storage account will be created as part of the deployment. If you set this to false, you can change it to true after deployment and add the storage account value. More info on the storage setting, check [Bulk Export](https://github.com/microsoft/fhir-server/blob/master/docs/BulkExport.md). Note that you will need owner or user-administrator permissions for this. If you encounter any storage permission or role assignment related issue, a quick workaround is that you set the option to false and ask your administrator to help update the setting later.|
+|Solution Type|**Yes**|Chose between deploying a Cosmos DB or SQL backed server|
+|Cosmos DB Account Consistency Policy|For Cosmos DB|An object representing the default consistency policy for the Cosmos DB account. See the [Cosmos DB Documentation](https://docs.microsoft.com/en-us/azure/templates/microsoft.documentdb/databaseaccounts#ConsistencyPolicy)|
+|Cosmos DB Free Tier|No|Whether to deploy using the Cosmos DB Free Tier Sku. There is a limit of one free Cosmos DB per subscription.|
+|Cosmos DB BYOK Key Url|No|The url for a customer managed Cosmos DB encryption key. If not provided a system managed key will be used. The provided key must be from an Azure Key Vault set up as described in the [documentation](https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-setup-cmk#configure-your-azure-key-vault-instance)|
```

Comment:
nit:  internally, we refer to this feature as "BYOK".  But, in public-facing code and documentation we use the more correct term: CMK (customer-managed key)